### ISSUE TITLE
Can I use FFmpeg in my app for free or i get a lawsuit ?

### ISSUE THREAD

**Issue by dubeyteja** [tanersener/flutter-ffmpeg#204]
I'm using FFmpeg for mixing audio and video and adding watermarks. I don't know much about LGPL V3.0 licence i just want to know can i use it or i will get any issues legally ?

**Comment by tanersener** [comment]
See [Patents](https://github.com/tanersener/mobile-ffmpeg/wiki/Patents) and [License](https://github.com/tanersener/mobile-ffmpeg/wiki/License) wiki pages of the parent `mobile-ffmpeg` project.

**Comment by dubeyteja** [comment]
the licence page clearly states that mobile-ffmpeg is under LGPL 3.0 license. So can i use it without using --enable-gpl or -gpl for free or is there anything i need to be aware of ?

**Comment by tanersener** [comment]
Don't ignore the patents, patent and license are different things. Read those two wiki pages carefully. I'm not a lawyer and I don't know when you get a lawsuit. Read [Using MobileFFmpeg in Commercial Applications](https://github.com/tanersener/mobile-ffmpeg/wiki/Using-MobileFFmpeg-in-Commercial-Applications) wiki page too.

**Comment by dubeyteja** [comment]
so I am using static LGPL libraries as I don't know how to use dynamic libraries and let the user to replace with the version he wants ? I got stuck here. I stopped my app development. if I am using static libraries I have give my source code out which I can't do. Is there any solution for this ?
I'm using flutter_ffmpeg for mixing audio and video ,add filters, adding watermarks, adding slow-motion, reverse to the video, merging multiple videos with audio provided by the user. or Can I use these for these purposes ?
